collectors/CollectorUp.py

```diff
@@ -0,0 +1,28 @@
+exp import os
+from prometheus_client.core import GaugeMetricFamily
+
+
+class CollectorUp:
+    def __init__(self, collectors, metrics=None):
+        self.collectors = list()
+        for collector in collectors:
+            self.collectors.append(collector[0])
+        self.metrics = metrics
+
+    def collect(self):
+        if os.environ['DEBUG'] >= '1':
+            print("Creating collector_up metrics")
+
+        g = GaugeMetricFamily('vrops_collector_up', 'testtext', labels=['collector'])
+
+        if not self.metrics:
```

Comment:
that's the `return False` from get_collector_up_information, nice!

Reply:
> What happens at the begin of the run? All collectors would first report 0, then 1, once they start delivering metrics? Shouldn't we give them a little time to avoid that?

No, the metrics are not generated until all collectors have run through once.